linkerd/admin/src/main/scala/io/buoyant/linkerd/admin/AdminHandler.scala

```diff
@@ -29,7 +29,7 @@ object AdminHandler extends HtmlView {
             <li><a href="/metrics">metrics</a></li>
             <li><a href="/admin/logging">logging</a></li>
             <li><a href="https://linkerd.io/help/">help</a></li>
-            <!-- <li><a href="/dashboard">Beta</a></li> -->
+            <li><a href="/dashboard">Beta dashboard</a></li>
```

Comment:
This is served on all the legacy admin pages (dtabs, metrics, etc).  Maybe this should be a link to the legacy dashboard?  Users can already return to the new dashboard by clicking on the logo (which links to "/").